__tests__/identity.test.js

```diff
@@ -262,4 +263,27 @@ describe('Tozny identity client', () => {
       approval,
     ])
   })
+  it('it can list groups with enabled mpc', async () => {
+    const reason = 'Reasons' + uuidv4()
+    const realmName = realmConfig.realmName
+    const accessControlledGroup = {
+      id: testTozIDGroupName,
+    }
+    const accessDurationSeconds = 1000
+    const createdAccessRequest = await ops.createAccessRequest(
+      realmConfig,
+      identity,
+      realmName,
+      [accessControlledGroup],
+      reason,
+      accessDurationSeconds
+    )
```

Comment:
this test might not need to create any access request data?
i _think_ the above code can be removed